src/app/workspace/results/list-view/list-view.component.scss

```diff
@@ -28,6 +28,20 @@
   }
 }
 
+.no-results-container {
+  background: rgb(255, 250, 230);
```

Comment:
I would prefer that we use hex values

Reply:
done in https://github.com/dasch-swiss/dsp-app/pull/871/commits/6aea59c9ee74f6b7e91a357050339aeb0032c5cd